### PR TITLE
[Qwik] match homepage style to other kits

### LINKED ISSUE #665: [qwik-pod] P1: Match Homepage/Page Styles To Other Kits

### PR DESCRIPTION
**Pull request by vyktoremario**
## Type of change

- [ ] Documentation change
- [x] Bug fix

## Summary of change

Screenshot 2022-12-16 at 17 07 10

## Checklist

- [ ] These changes follow the [contributing guidelines](https://github.com/thisdot/starter.dev/blob/main/CONTRIBUTING.md)
- [ ] This starter kit has been approved by the maintainers
- [ ] Changes for this new starter kit are being pushed to an integration branch instead of main
- [x] All dependencies are version locked
- [x] This fix resolves #665 
- [x] I have verified the fix works and introduces no further errors

### CODE REVIEW

**Review by SDaian — APPROVED** [review]
LGTM

File: starters/qwik-graphql-tailwind/src/routes/homepage.classNames.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export const container = 'text-center';
-export const h1Text = 'text-4xl font-bold';
-export const ul = 'list-inside list-disc my-8 text-xl';
-export const li = 'py-2';
-export const link = 'text-blue-500 underline italic hover:text-blue-800';
+export const header =
```
Comment (honzikec) [inline starters/qwik-graphql-tailwind/src/routes/homepage.classNames.ts:2]: I think we will have to move the classes out of separate files, but let's keep it in a separate PR (there should be a ticket for that).

**Review by honzikec — APPROVED** [review]
LGTM, just left an fyi comment 🚢
